website/source/docs/job-specification/service.html.md

```diff
@@ -355,6 +362,33 @@ service {
 }
 ```
 
+### gRPC Health Check
+
+gRPC health checks use the same host and port behavior as `http` and `tcp`
+checks, but gRPC checks also have an optional gRPC service to health check. Not
+all gRPC applications require a service to health check. gRPC health checks
+require Consul 1.0.5 or later.
+
+```hcl
+service {
+  check {
+    type            = "grpc"
+    port            = "rpc"
+    interval        = "5s"
+    timeout         = "2s"
+    grpc_service    = "grpc.health.v1.Health"
+    grpc_use_tls    = true
+    tls_skip_verify = true
+  }
+}
+```
+
+This check would translate to having a Consul check registration with the
+[GRPC][consul_grpc] parameter similar to `10.0.3.1:4567/grpc.health.v1.Health`.
```

Comment:
its not clear what "parameter similar to 10.0.3.1:4567/grpc.health.V1.health" means here

Reply:
Consul only has a single grpc field that takes the full `<host>:<port>/<endpoint>` like a URL. Nomad can't do that because the host and port are determined at runtime.

So I'm trying to describe how Nomad's fields relate to the Consul field, but yeah... this is worded badly. Perhaps the fact that it's a single `grpc` field in Consul is an implementation detail I don't actually have to worry about comparing against?

:thinking: I'll get this changed or remove it, but let me know if you have any suggestions!